test/store.test.js

```diff
@@ -172,13 +163,22 @@ test('store protocol integration flow', async t => {
 
   // Verify w3link can resolve uploaded file
   console.log('Checking w3link can fetch root', root.toString())
-  const w3linkResponse = await fetch(
-    `https://${root}.ipfs-staging.w3s.link`,
-    {
-      method: 'HEAD'
+  const gatewayURL = `https://${root}.ipfs-staging.w3s.link`
+  const gatewayRetries = 5
+  for (let i = 0; i < gatewayRetries; i++) {
```

Comment:
Since the CAR has to be indexed from a bucket event it may take a while to appear at the gateway. Adding retries with a short timeout seems to work.